logdna/logdna.py

```diff
@@ -256,6 +258,14 @@ def send_request(self, data):  # noqa: max-complexity: 13
                         'Error Response: %s', response.text)
                 return True  # discard
 
+            if status_code in [429]:
```

Comment:
it'd be cheaper to just do the direct comparison `status_code == 429` rather that constructing a new list every time.